adafruit_epd/il0373.py

```diff
@@ -67,7 +50,7 @@ class Adafruit_IL0373(Adafruit_EPD):
     def __init__(
         self, width, height, spi, *, cs_pin, dc_pin, sramcs_pin, rst_pin, busy_pin
     ):
-        super(Adafruit_IL0373, self).__init__(
```

Comment:
I'm not sure about removing these super arguments. I guess this would be something new with Black, and perhaps they are redundant, and therefore removed?

I'm not super familiar with Pythonic OOP yet I don't know if it's really safe to remove them.

Reply:
I found the documented pylint rule for this here: 
![image](https://user-images.githubusercontent.com/2406189/105428731-9b081980-5c15-11eb-8a3e-5230e8a09103.png)